src/sentry/replays/consumers/recording/process_recording.py

```diff
@@ -59,13 +60,14 @@ def __init__(
         self.__commit_data: MutableMapping[Partition, Position] = {}
         self.__last_committed: float = 0
 
+    @metrics.wraps("replays.process_recording.store_recording")
     def _store(
```

Comment:
is this function needed anymore? can `_process_recording` just call `ingest_recording_chunked` directly?